### PR TITLE
emit an event when a version is set to active in a db

### PR DESCRIPTION
**Pull request by jerry-stripe**
**Summary**
Emit a metric when a version is set to active in a db.

**Motivation**
This is a feature that has been asked for an is probably a useful thing to emit.

r? @scottjab-stripe 
cc? @stripe/storage

### CODE REVIEW

File: db.go [hunk 1]
```diff
@@ -203,6 +206,14 @@ func (db *db) upgrade(version *version) {
 	db.mux.upgrade(version)
 	version.setState(versionActive)
 
+	if version.stats != nil {
+		title := fmt.Sprintf("A new version is active in %s.", db.name)
+		text := fmt.Sprintf("Version %s has been set to active in db %s.", version.name, db.name)
+		event := statsd.NewEvent(title, text)
```
Comment (scottjab-stripe) [inline db.go]: Does this honor the config to override the statsd url?

Comment (jerry-stripe) [reply db.go]: It should, the url is set when [the client is created](https://github.com/stripe/sequins/blob/56fc7e96de1dee46ce50a3d70aedb7680b96c102/sequins.go#L73-L81). I'll also just manually confirmed

Comment (jerry-stripe) [reply db.go]: Whoops meant to say that I did manually confirm.

**Comment by scottjab-stripe** [comment]
LGTM

**Pull request merged**